website/other/bezier-rs-demos/src/main.ts

```diff
@@ -32,31 +17,63 @@ declare global {
 	}
 }
 
+window.document.title = "Bezier-rs Interactive Documentation";
+
 window.customElements.define("bezier-demo", BezierDemo);
 window.customElements.define("bezier-demo-pane", BezierDemoPane);
 window.customElements.define("subpath-demo", SubpathDemo);
 window.customElements.define("subpath-demo-pane", SubpathDemoPane);
 
-const bezierDemos = document.getElementById("bezier-demos");
-(Object.keys(bezierFeatures) as BezierFeatureName[]).forEach((featureName) => {
+function renderBezierPane(featureName: BezierFeatureKey, container: HTMLElement | null): void {
 	const feature = bezierFeatures[featureName];
 	const demo = document.createElement("bezier-demo-pane");
 
 	demo.setAttribute("name", featureName);
 	demo.setAttribute("demoOptions", JSON.stringify(feature.demoOptions || {}));
 	demo.setAttribute("triggerOnMouseMove", String(feature.triggerOnMouseMove));
 	demo.setAttribute("chooseComputeType", String(feature.chooseComputeType));
-	bezierDemos?.append(demo);
-});
+	container?.append(demo);
+}
 
-const subpathDemos = document.getElementById("subpath-demos");
-(Object.keys(subpathFeatures) as SubpathFeatureName[]).forEach((featureName) => {
+function renderSubpathPane(featureName: SubpathFeatureKey, container: HTMLElement | null): void {
 	const feature = subpathFeatures[featureName];
 	const demo = document.createElement("subpath-demo-pane");
 
 	demo.setAttribute("name", featureName);
 	demo.setAttribute("sliderOptions", JSON.stringify(feature.sliderOptions || []));
 	demo.setAttribute("triggerOnMouseMove", String(feature.triggerOnMouseMove));
 	demo.setAttribute("chooseComputeType", String(feature.chooseComputeType));
-	subpathDemos?.append(demo);
-});
+	container?.append(demo);
+}
+
+const pathname = window.location.pathname;
+const splitPathName = pathname.split("/");
+
+// Render based on pathname
+if (splitPathName[1] === "bezier" && splitPathName[2] in bezierFeatures) {
+	window.document.body.innerHTML = `<div id="bezier-demos"></div>`;
+	renderBezierPane(splitPathName[2] as BezierFeatureKey, document.getElementById("bezier-demos"));
+} else if (splitPathName[1] === "subpath" && splitPathName[2] in subpathFeatures) {
+	window.document.body.innerHTML = `<div id="subpath-demos"></div>`;
+	renderSubpathPane(splitPathName[2] as SubpathFeatureKey, document.getElementById("subpath-demos"));
+} else {
+	window.document.body.innerHTML = `
+  <h1>Bezier-rs Interactive Documentation</h1>
+  <p>
+    This is the interactive documentation for the <a href="https://crates.io/crates/bezier-rs"><b>Bezier-rs</b></a> library. View the
+    <a href="https://docs.rs/bezier-rs/latest/bezier_rs">crate documentation</a>
+    for detailed function descriptions and API usage. Click and drag on the endpoints of the demo curves to visualize the various Bezier utilities and functions.
+  </p>
+  
+  <h2>Beziers</h2>
+  <div id="bezier-demos"></div>
+  <h2>Subpaths</h2>
+  <div id="subpath-demos"></div>
```

Comment:
Feature request in to be added to the demos: include a `#` link after the name header for each demo so it can be clicked to change the URL hash fragment to that. Similar to many blogs that have that for different headings within an article. (In the respective demo file(s), not these four highlighted lines, of course.)